components/event-bus/internal/knative/subscription/controller/eventactivation/eventactivation_test.go

```diff
@@ -0,0 +1,125 @@
+package eventactivation
+
+import (
+	"fmt"
+	controllertesting "github.com/knative/eventing/pkg/controller/testing"
```

Comment:
The testing API used here is internal Knative testing API. In future releases, this can change or break. This is a conscious risk taken by the team since it provides the opportunity to write better tests.